content/docs/querying/operators.md

```diff
@@ -5,7 +5,13 @@ sort_rank: 2
 
 # Operators
 
-## Arithmetic binary operators
+## Binary operators
+
+Prometheus' query language supports the basic logical and arithmetic operators.
```

Comment:
Though it's not totally clear cut, we say "Prometheus's" for possession (http://grammar.ccc.commnet.edu/grammar/possessives.htm).

I don't know if I'd include a "the" in "the basic logical ...". It might confuse some readers ("the? which ones?"). Maybe just "supports basic logical..."?